### ISSUE TITLE
Question about CEN

### ISSUE THREAD

**Issue by cjs123456** [yikaiw/CEN#12]
Hello, thank you very much for the code you shared, we used your algorithm to train the model (aerial RGB and elevation modality, using ImageNet pre-trained model), but during model validation we found that the scaling factors of the two modalities are very close, and with the deepening of the convolution, the two modalities scaling factors of all channels are almost the same, the effect of exchange is not obvious.
May I ask where is the problem? 
Thanks in advance!

**Comment by yikaiw** [comment]
Hi, thanks for your interest. Have you applied the sparsity constraints (L1-norm) on the scaling factors? And the sparsity constraints should be applied individually on _disjoint_ parts of channels (as shown in Figure 2).

**Comment by cjs123456** [comment]
Thank you very much for the quick response! I’m sure the sparsity constraints (L1-norm) are applied on the scaling factors(as shown in Figure 2). But the result of the scaling factors visualization is shown in the following figure, two colors represent different modalities, which are very close
![image](https://user-images.githubusercontent.com/52772958/156287391-1b5b20de-260f-4251-9c95-447d89f333ff.png)

**Comment by yikaiw** [comment]
In this figure, it seems the sparsity constraints have little effect on the scaling factors, which is abnormal. Have you tried to increase lambda (in Eq. (4))?

**Comment by cjs123456** [comment]
Thank you very much for your great suggestion, The scaling factors visualization became reasonable after I tried to increase lamda. It seems like the sparsity constraints with higher lamda increases the difference of scaling factors between different modalities and highlights the non-negligible channels, which was discussed in detail in the paper [33](Learning Efficient Convolutional Networks through Network Slimming). 
I also want to ask a question, have you tried using the ratio of the scaling factors to decide whether to exchange channels? i.e. use ratio instead of threshold, such as exchange channels if the ratio of the scaling factors is higher than 0.2. I did some experiments like this but it didn't work very well, may I ask what is the problem?
Thank you very much！
